features/fixtures/shared/scenarios/Scenario.m

```diff
@@ -218,28 +213,29 @@ + (void)clearPersistentData {
 }
 
 + (void)executeMazeRunnerCommand:(void (^)(NSString *scenarioName, NSString *scenarioMode))preHandler {
-    NSLog(@"%s", __PRETTY_FUNCTION__);
+    logInfo(@"%s", __PRETTY_FUNCTION__);
```

Comment:
Same for me here - worthy of logging, for sure, but breaks the level of abstraction that I had been aiming for.